init/src/init.ts

```diff
@@ -350,17 +350,16 @@ html {
     // Skip this and be silent if there is a network issue.
   }
 
-  const MAIN_TSX = `import { App, staticFiles, fsRoutes } from "@fresh/core";
+  const MAIN_TS = `import { App, staticFiles, fsRoutes } from "@fresh/core";
 import { State } from "./utils.ts";
 
-export const app = new App<State>()
-  .use(staticFiles())
-  .get("/api/:joke", () => new Response("Hello World"))
-  .get("/greet/:name", (ctx) => {
-    return ctx.render(<h1>Hello {ctx.params.name}</h1>);
```

Comment:
No more JSX rendering?

Reply:
It made the file a tsx file, which was annoying. Most users will not be doing jsx in their main file

Reply:
ok shots fired.